authorizations.md

```diff
@@ -0,0 +1,93 @@
+# Authorizations Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+An authorizations is the representation of some rights that are available to a specific user (eperson) on a defined object, eventually the whole repository (site object).
+All the authorizations are always explicitly listed regardless to how they are grant, by direct policies via groups' membership or open to everyone (anonymous users).
+Please note that these endpoints are all READ-ONLY, authorizations are granted and withdrawn as result of business processes, configurations and resource policies.
+
+## Main Endpoint
+**/api/authz/authorizations**
```

Comment:
I'm not sure exactly what this endpoint might retrieve. I think it would be useful to have an example here.
Also, if this retrieves a list of "my" authorizations, should I assume a nonexistent authorization is a DENIED one?

Reply:
well nothing :) 
the /api/authz/authorizations is not implemented and will return a 405 code as stated in the contract.

Keep in mind the general approach that we have adopted for our REST api. Each endpoints represent a collection of resources. The GET method over the collection should return, where it makes sense, all the resources in the collection. Don't be confused by the term collection here, it is referred to the REST side, see 
https://github.com/DSpace/Rest7Contract/blob/master/README.md#use-of-the-http-verbs-and-http-response-code

About the "denied" authorizations: yes, we keep the general dspace principle here. If you don't have an explicit authorization then it is denied. So, assuming that you want to know if an user is authorized or not to withdraw an Item you can use
`/api/authz/authorizations/search/object?uri=https://dspace.url/api/core/items/<item-uuid>&eperson=<eperson-uuid>&feature=withdrawItem`
if you get a not empty answer, i.e. you have a matching authorization, the user is allowed otherwise he is denied

Reply:
Throwing a `405 Method Not Allowed` to all requests on this endpoint seems appropriate for now.